### PR TITLE
Bug 2000590: Warning on topology context menu right click

### PR DESCRIPTION
**Pull request by yozaam**
**Fixes**: 

https://issues.redhat.com/browse/ODC-6105

**Analysis / Root cause**: 

- ref was getting added to `KebabItem` function component

**Solution Description**: 

- wrapped the function component with a React Fragment `<> </>`

**Screen shots / Gifs for design review**: 

No warning is seen anymore
image

**Unit test coverage report**: 

**Test setup:**

Open topology.
Right click on any workload.

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by yozaam** [comment]
/assign @rohitkrai03 
requesting approval

File: frontend/packages/topology/src/components/graph-view/components/nodeContextMenu.tsx [hunk 1]
```diff
@@ -46,7 +46,11 @@ export const createMenuItems = (actions: KebabMenuOption[]) =>
     ) : (
       <ContextMenuItem
         key={index} // eslint-disable-line react/no-array-index-key
-        component={<KebabItem option={option} onClick={() => onKebabOptionClick(option)} />}
+        component={
+          <>
```
Comment (rohitkrai03) [inline frontend/packages/topology/src/components/graph-view/components/nodeContextMenu.tsx:50]: Can you add a comment here why we need to wrap this in fragments?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

**Review by rohitkrai03 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**